README.md

```diff
@@ -1 +1,50 @@
-# Best-Webdev-Practice
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List Application
+
+> "To-Do-List App" is a simple web app that displays a list of daily tasks and allows you to edit, add and remove task from that list.
+
+## Built With (Linter)
+
+- Webpack
+- HTML (Webhint)
+- CSS (Stylelint)
+- JavaScript (Eslint)
+
```

Comment:
Optional: 😄 
You can add some screenshots to make you app accessible

Reply:
Thanks, I added a screenshot of the application